tests/pytorch/test_models_zoo_pt.py

```diff
@@ -1,9 +1,46 @@
 import numpy as np
 import pytest
+from test_models_detection_pt import test_detectionpredictor_pt  # noqa: F401
+from test_models_recognition_pt import test_recognitionpredictor_pt  # noqa: F401
```

Comment:
Let's just recreate the detection & recognition predictors directly here :ok_hand: